src/Umbraco.Tests/Services/ContentTypeServiceTests.cs

```diff
@@ -1166,19 +1166,20 @@ public void Cannot_Rename_PropertyGroup_On_Child_Avoiding_Conflict_With_Parent_P
             {
                  Name = "Author", Description = "",  Mandatory = false, SortOrder = 1, DataTypeId = -88
             };
-            var subtitleAdded = contentPage.AddPropertyType(subtitlePropertyType, "Content");
-            var authorAdded = contentPage.AddPropertyType(authorPropertyType, "Content");
+            var subtitleAdded = contentPage.AddPropertyType(subtitlePropertyType, "content", "Content");
+            var authorAdded = contentPage.AddPropertyType(authorPropertyType, "content", "Content");
             service.Save(contentPage);
 
             var compositionAdded = contentPage.AddContentType(contentMetaComposition);
             service.Save(contentPage);
 
-            //Change the name of the tab on the "root" content type 'page'.
-            var propertyGroup = contentPage.PropertyGroups["Content_"];
-            Assert.Throws<Exception>(() => contentPage.PropertyGroups.Add(new PropertyGroup(true)
+            // Change the name of the tab on the "root" content type 'page'
+            var propertyGroup = contentPage.PropertyGroups["Content 2"];
+            Assert.Throws<ArgumentException>(() => contentPage.PropertyGroups.Add(new PropertyGroup(true)
             {
                 Id = propertyGroup.Id,
                 Name = "Content",
+                //Alias = "content",
```

Comment:
For no real reason 🙃 It would otherwise be automatically generated when adding the property group: https://github.com/umbraco/Umbraco-CMS/blob/b6d4cc467e04b7fffb5c5eb2026293ac2c511895/src/Umbraco.Core/Models/PropertyGroupCollection.cs#L105-L109